packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -505,5 +505,14 @@ export class FeatureFlags {
         defaultValueForNewProjects: false,
       },
     ]);
+
+    this.registerFlag('auth', [
+      {
+        name: 'enableCaseInsensitivity',
```

Comment:
it's kinda confusing that the FF is "case insensitive" but the CFN value is "case sensitive". I'd recommend naming the FF "isUsernameCaseSensitive"

Reply:
This aligns with how it is named in the Cognito Console. That's the reason for the naming, I agree that comparing CFN and this counter intuitive, but in the Cognito Console it is like this.